stan/math/rev/core/vari.hpp

```diff
@@ -747,9 +676,6 @@ class vari_value<T, require_eigen_sparse_base_t<T>> : public vari_base,
   friend class var_value;
 };
 
-// For backwards compatability the default is double
-using vari = vari_value<double>;
```

Comment:
Why is this deleted?

Reply:
It's moved up to be defined right after the scalar vari_value on line 172